layer/logged_data/retrying_files_sender.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+import requests  # type: ignore
+
+
+class RetryingFilesSender:
```

Comment:
nit: I'd probably call this `FileUploader`, and rename `store` to `upload`. (I'd skip the "Retrying" part since we have one such class)

Reply:
renamed